UnitTests/UnitTests.HighPerformance.NetCore/UnitTests.HighPerformance.NetCore.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp2.1;netcoreapp3.0</TargetFrameworks>
+    <LangVersion>8.0</LangVersion>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <!-- .NET Core 2.1 doesn't have the Unsafe type -->
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netcoreapp2.1' ">
+    <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="4.7.0" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.2.0" />
```

Comment:
No particular reason, I think those were just the packages that Visual Studio added automatically when I created that unit test project. Fixed this in https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3128/commits/570b083db52e29fb6af02f5edbe587558e5d7413. 😄

I didn't update the references in the UWP project as the Test SDK one is locked since it's imported from outside the project itself, so I guess that'll be updated once #3165 is merged. Should I still update at least the adapter/framework packages in there as well anyway?